test/Microsoft.ML.Functional.Tests/ONNX.cs

```diff
@@ -109,7 +104,7 @@ public void SaveOnnxModelLoadAndScoreKMeans()
             // TODO #2980: ONNX outputs don't match the outputs of the model, so we must hand-correct this for now.
             // TODO #2981: ONNX models cannot be fit as part of a pipeline, so we must use a workaround like this.
```

Comment:
Are these comments still valid? Do we do hand correction now?

Reply:
The comments are not valid. I have removed them in the next commit. 

---
In reply to: [381300021](https://github.com/dotnet/machinelearning/pull/4849#discussion_r381300021) [](ancestors = 381300021)